lua/wikis/commons/HiddenDataBox.lua

```diff
@@ -194,15 +194,15 @@ end
 ---Validates the provided tier, tierType pair
 ---@param tier string|number|nil
 ---@param tierType string?
----@return string|number|nil, string?, string[]
+---@return integer?, string?, string[]
 function HiddenDataBox.validateTier(tier, tierType)
 	local warnings = {}
 
 	if not tier and not tierType then
 		return nil, nil, warnings
 	end
 
-	local tierValue, tierTypeValue = Tier.toValue(tier, tierType)
+	local tierValue, tierTypeValue = Tier.toValue(tier --[[@as string|integer]], tierType)
```

Comment:
This doesn't look true? If tier input is nil, and tierType is non-nil, then the first param in the call will be nil

Reply:
this is only because of smash like wikis ...
every other wiki will always have tier set